include/macros.h

```diff
@@ -11,6 +11,9 @@
 #define SCREEN_WIDTH_HIGH_RES  576
 #define SCREEN_HEIGHT_HIGH_RES 454
 
+#define SCREEN_TO_DEVICE_X(projectedPos, invW) ((projectedPos).x * (invW) * (SCREEN_WIDTH / 2) + (SCREEN_WIDTH / 2))
+#define SCREEN_TO_DEVICE_Y(projectedPos, invW) ((projectedPos).y * (invW) * (-SCREEN_HEIGHT / 2) + (SCREEN_HEIGHT / 2))
```

Comment:
From my understanding, these macros goes against the current naming scheme of the oot/mm repos. I'm not necessarily opposed to the changed, but if I understand correctly, this makes the assumption that:
`projectionPos` is renamed to `screenPos`
`screenPos`  is renamed to `devicePos`


`projectionPos`/`screenPos` has the advantage of being more intuitive and more standard (projectionPos based on projectionMtx), and seems easier to understand from users (both projection and screen positions make sense to me, screen/device did not make intuitive sense at first).
Plus, everyone is used to screen coordinates being (0, SCREEN_WIDTH), instead of what you propose here (-1, 1)

On the other hand, there does seem to be consistency with Nintendo for the change you're proposing:
https://ultra64.ca/files/documentation/online-manuals/man-v5-2/allman52/kantan/step3/1/1_3.htm

Many things would need to change to follow this convention you're proposing (all the projected pos, `SCREEN_XXX` macros, it could quickly add up to 1000's of lines)

I don't feel the need to follow Ninentdo's naming convension at all if there's better and less confusing names. but I would be open to hearing others feedback, and I think this is a bigger discussion to be had instead of tacking on to the end of a PR

Reply:
I'll put it back to `PROJECTED_TO_SCREEN` for now, then.